packages/hardhat/hardhat.config.ts

```diff
@@ -10,14 +10,13 @@ import "@nomicfoundation/hardhat-verify";
 import "hardhat-deploy";
 import "hardhat-deploy-ethers";
 
-// If not set, it uses ours Alchemy's default API key.
-// You can get your own at https://dashboard.alchemyapi.io
-const providerApiKey = process.env.ALCHEMY_API_KEY || "oKxs-03sij-U_N0iOlrSsZFr29-IqbuF";
 // If not set, it uses the hardhat account 0 private key.
 const deployerPrivateKey =
   process.env.DEPLOYER_PRIVATE_KEY ?? "0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80";
 // If not set, it uses ours Etherscan default API key.
 const etherscanApiKey = process.env.ETHERSCAN_API_KEY || "DNXJA8RX2Q3VZ4URQIWP7Z68CJXQZSC6AW";
+// forking rpc url
+const forkingURL = process.env.FORKING_URL || "";
```

Comment:
won't the mainnet one work? `https://cloudflare-eth.com`

Or does it need to be a special RPC?

(Trying to understand why we are using an env var for this)

Reply:
![Screenshot 2024-10-10 at 9 05 07 AM](https://github.com/user-attachments/assets/885c426d-7171-4f3d-84c6-46565e7f53c4)

Public rpc are bad forking and most of the time doesn't work. The above SS is for `https://cloudflare-eth.com`. 

![Screenshot 2024-10-10 at 8 59 34 AM](https://github.com/user-attachments/assets/ad4b2b09-4ccd-4fc1-aeef-3df270e01eed)

The above is `eth.llama.rpc` and it didn't work too :( 

And also mentioned here https://github.com/scaffold-eth/scaffold-eth-2/pull/955#discussion_r1793305774 it might  be the case that the `https://cloudflare-eth.com` block the IP and then frontend will not work too because it uses the same RPC URL :(